rst_files/julia_arrays.rst

```diff
@@ -268,41 +212,32 @@ In such cases, we don't care about the distinction between row and column vector
 This is why many Julia functions return flat arrays by default
 
 
-
-
-
 .. _creating_arrays:
 
 Creating Arrays
 ------------------
 
 
-
 Functions that Return Arrays
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 We've already seen some functions for creating arrays
 
 .. code-block:: julia
 
-    eye(2)
-
-
+    Matrix{Float64}(I, 2, 2)
 
 .. code-block:: julia
 
     zeros(3)
 
 
-
-
 You can create an empty array using the ``Array()`` constructor
 
 .. code-block:: julia
 
-    x = Array{Float64}(2, 2)
+    x = Array{Float64}(undef, 2, 2)
```

Comment:
This is a direct port of the relevant QuantEcon...but I still think it could be confusing. Will defer to @jlperla.

Reply:
I think we can hold off on most stylistic changes for now.

Reply:
OK, got it.

Reply:
I kept that one since it is teaching how to use the constructor. The next lines go about how those are garbage values, etc.